kythe/go/extractors/config/wrapper/gradle_build_modifier_test.go

```diff
@@ -0,0 +1,123 @@
+package wrapper
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+const testDataDir = "testdata"
+
+var multipleNewLines = regexp.MustCompile("\n{2,}")
+
+func filesEqual(got, want []byte) (bool, string) {
```

Comment:
This is now the second or third or so file where we have this copied in.  Wonder if we should make a test util for it.  #2860 to track